### PR TITLE
chore: rename default branch to `main`

### PR DESCRIPTION
**Pull request by hairmare**
# Description

Once in a while, some housekeeping makes sense:

* renames default branch to `main`
* get rid of all currently deprecated charts
* update helm-docs usage to new short-form

I've been holding off on the `s/master/main/` because i was afraid of it breaking open pull requests, this isn't an issue with github anymore nowadays.

Once approved, I'll rename the branch via the github interface right before i merge this.

# Issues

# Checklist

* [x] This PR contains a description of the changes I'm making
* [x] I updated the version in Chart.yaml
* [x] I updated applicable README.md files using  `pre-commit run`
* [x] I documented any high-level concepts I'm introducing in `docs/`
* [x] CI is currently green and this is ready for review
* [x] I am ready to test changes after they are applied and released

### CODE REVIEW

File: charts/back8sup/README.md [hunk 1]
```diff
@@ -12,29 +12,16 @@ This chart is maintained by [Adfinis](https://adfinis.com/?pk_campaign=github&pk
...
-| cronjob.environment[2].value | string | `"/var/run/secrets/kubernetes.io/serviceaccount/token"` |  |
-| cronjob.environment[3].name | string | `"CONFIGMAP_PATH"` |  |
-| cronjob.environment[3].value | string | `"/etc/config.yaml"` |  |
-| cronjob.environment[4].name | string | `"EXPORT_FORMAT"` |  |
-| cronjob.environment[4].value | string | `"yaml"` |  |
-| cronjob.environment[5].name | string | `"DST_FOLDER"` |  |
-| cronjob.environment[5].value | string | `"/mnt/back8sup"` |  |
-| cronjob.environment[6].name | string | `"GENERATIONS"` |  |
-| cronjob.environment[6].value | string | `"30"` |  |
+| cronjob.environment | list | `[{"name":"API_ENDPOINT","value":"https://kubernetes.default.svc.cluster.local:443"},{"name":"CA_CERT","value":"/var/run/secrets/kubernetes.io/serviceaccount/ca.crt"},{"name":"TOKEN_FILE","value":"/var/run/secrets/kubernetes.io/serviceaccount/token"},{"name":"CONFIGMAP_PATH","value":"/etc/config.yaml"},{"name":"EXPORT_FORMAT","value":"yaml"},{"name":"DST_FOLDER","value":"/mnt/back8sup"},{"name":"GENERATIONS","value":"30"}]` | specifies the different environment variable to configure the cronjob |
```
Comment (eyenx) [inline charts/back8sup/README.md:24]: does it make sense to have a description at this level?

File: charts/cert-manager-issuers/Chart.yaml [hunk 2]
```diff
@@ -2,14 +2,14 @@ apiVersion: v2
...
 # matches apiVersion: cert-manager.io/v1
 appVersion: "1"
 keywords:
   - cert-manager
   - tls
 home: https://cert-manager.io/docs/configuration/
-icon: https://raw.githubusercontent.com/adfinis-sygroup/helm-charts/master/charts/cert-manager-issuers/logo.png
+icon: https://raw.githubusercontent.com/adfinis-sygroup/helm-charts/main/charts/cert-manager-issuers/logo.png
 sources:
   - https://github.com/adfinis-sygroup/helm-charts
```
Comment (eyenx) [inline charts/cert-manager-issuers/Chart.yaml:14]:
```suggestion
  - https://github.com/adfinis-sygroup/helm-charts/tree/main/charts/cert-manager-issuers
```

File: charts/misc-apps/README.md [hunk 3]
```diff
@@ -27,7 +27,6 @@ This chart is maintained by [Adfinis](https://adfinis.com/?pk_campaign=github&pk
 | chartmuseum.chart | string | `"chartmuseum"` | Chart |
 | chartmuseum.destination.namespace | string | `"infra-chartmuseum"` | Namespace |
 | chartmuseum.enabled | bool | `false` | Enable chartmuseum |
-| chartmuseum.repoURL | string | `"https://chartmuseum.github.io/charts"` | [repo](https://chartmuseum.github.io/charts) |
```
Comment (eyenx) [inline charts/misc-apps/README.md:30]: seems like this broke

File: charts/misc-apps/README.md [hunk 4]
```diff
@@ -48,7 +47,6 @@ This chart is maintained by [Adfinis](https://adfinis.com/?pk_campaign=github&pk
 | prometheusMsteams.chart | string | `"prometheus-msteams"` | Chart |
 | prometheusMsteams.destination.namespace | string | `"infra-prometheus-msteams"` | Namespace |
 | prometheusMsteams.enabled | bool | `false` | Enable prometheus-msteams |
-| prometheusMsteams.repoURL | string | `"https://prometheus-msteams.github.io/prometheus-msteams"` | [repo](https://prometheus-msteams.github.io/prometheus-msteams) |
```
Comment (eyenx) [inline charts/misc-apps/README.md:51]: this broke too

File: charts/back8sup/README.md (hunk 1 above)
Comment (hairmare) [reply charts/back8sup/README.md]: i removed the comment on `cronjob.environment` to make helm-docs render the individual bits again in a9d2eb3ed882077f7fe0af036c0236775a65ad0e

File: charts/cert-manager-issuers/Chart.yaml (hunk 2 above)
Comment (hairmare) [reply charts/cert-manager-issuers/Chart.yaml]: a9d2eb3ed882077f7fe0af036c0236775a65ad0e

File: charts/misc-apps/README.md (hunk 4 above)
Comment (hairmare) [reply charts/misc-apps/README.md:51]: the repoURL was missing a description which i added in ff284f8a897b501bbd81fb6753984befdf98a0e8

File: charts/misc-apps/README.md (hunk 3 above)
Comment (hairmare) [reply charts/misc-apps/README.md:30]: same as prom teams thing ff284f8a897b501bbd81fb6753984befdf98a0e8

**Review by eyenx — APPROVED** [review]
Good JOB!

File: charts/argoconfig/Chart.yaml [hunk 5]
```diff
@@ -2,10 +2,10 @@ apiVersion: v2
 name: argoconfig
 description: Configure Argo CD AppProjects and Applications
 type: library
-version: 0.7.4
+version: 0.7.5
```
Comment (tongpu) [inline charts/argoconfig/Chart.yaml:5]: Do we have to update the dependency to the `argoconfig` chart in the `*-apps` or are we just going to ignore it, as it is not actually an upgrade?

Comment (hairmare) [reply charts/argoconfig/Chart.yaml:5]: we need to publish the new versions before we can update dependants

this change only really touches docs, so bumping anything that depends on `argoconfig` and `common` doesn't need a high priority as it is indeed a cosmetic change, i'm assuming we'll do it anyway nevertheless

Comment (tongpu) [reply charts/argoconfig/Chart.yaml:5]: Right, we can't bump until the new version is actually being published. Regarding the priority of bumping I'm totally with you.

**Review by tongpu — APPROVED** [review]

**Pull request merged**